scripts/aws/monitor.py

```diff
@@ -179,6 +185,22 @@ def get_topic_arn(client, topic_name):
     return None
 
 
+def enable_cloudwatchlogs_in_stages(client, gw_name, logging_level):
+    """enables the cloudwatch logs for all the stages"""
+    gw_res = client.get_rest_apis()
+    for value in gw_res['items']:
+        if value['name'] == gw_name:
```

Comment:
I see why unique names matter based on this lookup - what I don't understand is why this get-by-name lookup is necessary, instead of just looking up the gateway by its ID (as in, passing `gw_id` into `enable_cloudwatchlogs_in_stages` instead of `gw_name` which is not guaranteed to be unique and likely _won't_ be unique in production).